TheMovieDb/Modules/Searching/SearchingView.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  SearchingView.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import Foundation
+import UIKit
+
+final class SearchingView: UIViewController {
+
+    // MARK: Properties
+    var presenter: SearchingPresenterProtocol?
+    private var tableView: UITableView!
+    private let searchController = UISearchController(searchResultsController: nil)
+    private var viewModel: [MovieViewModel] = []
+
+    // MARK: Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        presenter?.viewDidLoad()
+        configureUI()
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        DispatchQueue.main.async {
+            self.searchController.searchBar.becomeFirstResponder()
+        }
+    }
+    
+    // MARK: Helpers
+    
+    private func configureUI() {
+        configureTableView()
+        configureSearchController()
+
+    }
+    
+    private func configureTableView() {
```

Comment:
trick: since you are having more than a single private func you can create a private extension and all methods there will be private ;)

Reply:
done!! 😁